lib/index.js

```diff
@@ -50,10 +50,10 @@ const Rooftop = {
       apiToken: opts.apiToken,
       contentType: (name) => {
         return {
-          get: (opts = {}) => {
-            const params = Object.keys(opts).join(',')
+          get: (opts = { params: {} }) => {
+            const params = Object.keys(opts.params).join(',')
```

Comment:
That's a good point, and in my opinion we don't want that (it'll cause errors). We should probably go with the below instead:

```javascript
          get: (opts = {}) => {
            opts.params = opts.params || {}
```

This lets a user provide an objects object with or without a params key. If a params object is provided it's used, if not a blank object is used instead.